### PR TITLE
RHCOS Post Install actions for scale-ci on AWS

### PR DESCRIPTION
**Pull request by akrzos**

### CODE REVIEW

**Comment by akrzos** [comment]
@chaitanyaenr Please review and merge, I believe this is ready at this point we can remove the CVO disable once that gets resolved.

File: rhcos-scale.yml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
   gather_facts: false
-  remote_user: core
+  remote_user: "{{orchestration_user}}"
   vars_files:
     - vars/rhcos-scale.yml
   tasks:
     - name: Get cluster name
       shell: |
-        {%raw%}oc get machineset -n openshift-cluster-api -o=go-template='{{(index (index .items 0 ).metadata.labels "sigs.k8s.io/cluster-api-cluster") }}'{%endraw%}
+        {%raw%}oc get machineset -n openshift-machine-api -o=go-template='{{(index (index .items 0 ).metadata.labels "sigs.k8s.io/cluster-api-cluster") }}'{%endraw%}
```
Comment (chaitanyaenr) [inline rhcos-scale.yml]: @akrzos the namespace changes are already present in the master branch.

Comment (akrzos) [reply rhcos-scale.yml]: Rgr, I have now parameter-ized this.

**Comment by akrzos** [comment]
Just tested on latest cluster and it is all working.  Pushing in so we can close out Jira Card.

**Pull request merged**